### PR TITLE
Skeletonize

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request skeletonizes this repo by pulling in the upstream changes from [cisagov/skeleton-generic](https://github.com/cisagov/skeleton-generic).

## 💭 Motivation and context ##

TravisCI is no longer free for open source and we have used up all our credits, so at a minimum this repo needed to be modified to use GitHub Actions.  Since I was going to be doing that I figured I would go ahead and bring it into the light as well.

## 🧪 Testing ##

All `pre-commit` hooks pass.  I didn't change the `docker-compose.yml` file in any substantial way, so I see no reason why this repo wouldn't continue to work as it did before.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
* [x] All new and existing tests pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,23 @@
-# NCATS Orchestrator :notes: :musical_note: #
+# orchestrator #
```
Comment (dav3r) [inline README.md]:
```suggestion
# Orchestrator #
```

File: start.sh [hunk 2]
```diff
@@ -1,8 +0,0 @@
-#!/bin/bash
-
-set -o nounset
-set -o errexit
-set -o pipefail
-
-cd /var/cyhy/orchestrator || exit 1
-docker-compose up -d
```
Comment (dav3r) [inline start.sh:8]: I know it's basic, but we should probably add a simple "Usage" section to the README for Docker Compose newbies.  For reference: https://github.com/cisagov/skeleton-docker#usage

**Review by dav3r — APPROVED** [review]
Excellent work, though I had a couple of small notes. 🌮

File: README.md [hunk 3]
```diff
@@ -28,17 +31,23 @@ various Docker containers to use.  These files are:
 
 As an example, `secrets/cyhy_read_creds.yml` should look something
 like this:
-```
+
+```yaml
 version: '1'
```
Comment (mcdonnnj) [inline README.md:37]:
```suggestion
---
version: '1'
```

File: .gitignore [hunk 4]
```diff
@@ -1,4 +1,3 @@
+__pycache__
+.mypy_cache
 .python-version
-secrets
```
Comment (mcdonnnj) [inline .gitignore:2]: Should `secrets` remain to align with the instructions to create such a directory in the README?

File: README.md [hunk 5]
```diff
@@ -1,20 +1,23 @@
-# NCATS Orchestrator :notes: :musical_note: #
+# Orchestrator #
```
Comment (mcdonnnj) [inline README.md:1]: RIP 🎶🎵

**Review by mcdonnnj — APPROVED** [review]
Thanks for taking care of this! Just a couple small suggestions 👍

File: start.sh (hunk 2 above)
Comment (jsf9k) [reply start.sh:8]: See commit 2a2c565.

File: README.md (hunk 5 above)
Comment (jsf9k) [reply README.md:1]: :cry:

File: .gitignore (hunk 4 above)
Comment (jsf9k) [reply .gitignore:2]: See commit 793d553.

**Review by dav3r — APPROVED** [review]
🥁

**Review by mcdonnnj — APPROVED** [review]
Even better the second time! :shipit:

**Pull request merged**